scripts/common.py

```diff
@@ -57,6 +57,13 @@
 # there is a 'make test' bug in redis 6.0.6 where the solution has not been
 # released. This is explained in this issue:
 # https://github.com/redis/redis/issues/7540.
+# When upgrading the redis cli version, make sure to run 'make test' on the
```

Comment:
Flag this so it stands out. Add something like "IMPORTANT NOTE TO DEVELOPERS WHO CHANGE THIS: " or "UPGRADE INSTRUCTIONS: " or something like that. Add a newline above line 56 and a line with "#" above this one.